source/components/organisms/RemoveCaseModal/RemoveCaseModal.tsx

```diff
@@ -52,7 +52,7 @@ function RemoveCaseModal({
     },
     [RemoveCaseState.Loading]: {
       text: {
-        title: "Ditt ärende tas bort",
+        title: "Ditt ärende rensas",
```

Comment:
You should use the same kind of words in the Loading state as you do in the Default. 

Change title and body in Loading to:

```js
{
  title: "Din ansökan tas bort",
  body: "Vänligen vänta ..."
}
```